effect/jvm/src/main/scala/scalaz/effect/RTS.scala

```diff
@@ -471,13 +471,14 @@ private object RTS {
                         curIo = nextInstr(value, stack)
 
                         if (curIo == null) {
+                          println("GOT HERE!!!!!!!!!!!!")
```

Comment:
heh?

Reply:
Hehe. Ignore, will delete in next commit. Tried to force that code path through tests (unsuccessfully).